account_invoice_show_currency_rate/tests/test_account_move.py

```diff
@@ -50,8 +51,8 @@ def setUpClass(cls):
             {"name": "Pricelist EUR", "currency_id": cls.currency_eur.id}
         )
         # Create custom rates to USD + EUR
-        cls._create_currency_rate(cls, cls.currency_usd, "2000-01-01", 1.0)
-        cls._create_currency_rate(cls, cls.currency_eur, "2000-01-01", 2.0)
+        cls._create_currency_rate(cls, cls.currency_usd, fields.Date.today(), 1.0)
```

Comment:
Can you avoid variable dates?

Reply:
It has been changed in order to standardize it and not need to define a date. Similar to what the https://github.com/OCA/purchase-workflow/blob/13.0/purchase_stock_picking_show_currency_rate addon does in tests.
Do you think it is better to define those specific dates?

Reply:
The approach is to have the less variable data possible. That's why we also avoid demo data. Some side effects derived from using variable dates: error in tests on leap years and around midnight.